src/common/preload-custom-css.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-var, func-names, vars-on-top */
```

Comment:
ESLint is not supposed to be disabled. I think we should achieve this by a more modern way.

Reply:
Yeah I agree but since I don't know anything about bundlers and how to make them compile the normal modern syntax into ES5, I had to write in ES5 manually, otherwise the minifier couldn't process the file.

Reply:
BTW if you think this is really crucial you may want to add an option to disable disabling eslint via the inline comments.